_episodes/05-starting-jekyll.md

```diff
@@ -7,75 +7,351 @@ questions:
+
+   ~~~
+    # {% raw %}{{ site.title }}{% endraw %}
+   
+    ## Description
+    {% raw %}{{ site.description }}{% endraw %}
+
+    Have any questions about what we do? [We'd love to hear from you!]({% raw %}mailto:{{ site.email }}{% endraw %})
+   ~~~  
+   
+2. We can use the same parameter in different pages. Let's reuse `{% raw %}{{ site.description }}{% endraw %}` and 
+`{% raw %}{{ site.email }}{% endraw %}` in `about.md` like this:
+
+    ~~~ 
+    # About
+   
+    ## Project
+    {% raw %}{{ site.description }}{% endraw %}
+           
+    ## Funders
+    We gratefully acknowledge funding from the XYZ Founding Council, under grant number 'abc'.
+   
+    ## Cite us
+    You can cite the project as:  
+    
+    >    *My research project. Zenodo. https://zenodo.org/record/doi*
+    
+    ## Contact us
+   
+    Email: [team@my.research.org](mailto:{% raw %}{{ site.email }}{% endraw %})   
+    
+    Twitter: [@my_research_project](https://twitter.com/my_research_project)   
+    ~~~
+    
+3. Go to your website to see the changes.
+4. Note that site parameters will not render nicely when viewing files in GitHub (they will be displayed as text
+`{% raw %}{{ site.PARAMETER_NAME }}{% endraw %}` rather than the parameter's rendered value) but will in the website.
+
+> ## Exercise 
+> In `about.md` we have a Twitter URL under the 'Contact us' section. That's one piece of information that could go into 
+> global parameters in `_config.yml` as you may want to repeat it on a footer of every page. 
+> Make changes to your website to extract Twitter URL as a global parameter.
 > > ## Solution
+> > 1. Add parameter twitter to `_config.yml`: 
 > >
-> > We will create the `pi` variable by adding the following line to the yaml header.
-> > ~~~
-> > pi: "Ada Lovelace"
-> > ~~~
+> >    ~~~
+> >    title: "Building Websites in GitHub"
+> >    description: "This research project develops training materials for reseachers wanting to learn to build project
+> >    websites in GitHub with GitHub Pages."
+> >    email: "team@my.research.org"
+> >    twitter: "https://twitter.com/my_research_project"
+> >    ~~~
+> >    {: .language-yaml}
+> > 2. Make use of the twitter parameter in `about.md`: 
 > >
-> > Here is an example of using it in the `index.md` file
-> > ~~~
-> > ---
-> > pi: "Ada Lovelace"
-> > ---
+> >    ~~~  
+> >    # About
+> >   
+> >    ## Project
+> >    {% raw %}{{ site.description }}{% endraw %}
+> >           
+> >    ## Funders
+> >    We gratefully acknowledge funding from the XYZ Founding Council, under grant number 'abc'.
+> >   
+> >    ## Cite us
+> >    You can cite the project as:  
+> >    
+> >    >    *My research project. Zenodo. https://zenodo.org/record/doi*
+> >    
+> >    ## Contact us
+> >   
+> >    Email: [team@my.research.org](mailto:{% raw %}{{ site.email }}{% endraw %})   
+> >    
+> >    Twitter: [@my_research_project]({% raw %}{{ site.twitter }}{% endraw %})   
```

Comment:
related to my concern above about making the link test the twitter handle instead of a variable.

Maybe the twitter variable should be 
```
twitter: "my_research_project"
```

and the entry in 

```suggestion
> >    Twitter: [@{% raw %}{{ site.twitter }}{% endraw %}](https://twitter.com/{% raw %}{{ site.twitter }}{% endraw %})   
```